### DISCUSSION TITLE
How to make one view to display the activities of all tenants in single (possibly different) App.

### DISCUSSION THREAD

**Discussion by nelson8013** [spatie/laravel-multitenancy#369] [Q&A] [answered]
I am building a school management application for schools and the Spatie Multi-tenant package has helped make the app tenant aware.

Now, I need to create an admin view that shows all schools (tenants), the number of parents in the school, number of students, number of teachers, etc.

Can anyone help?

Technical Details:
1. Database: Multiple Databases (one DB per tenant).
2. Spatie multi-tenant v1.

**Comment by masterix21** [comment]
What are your doubts about it?

**Reply by nelson8013** [reply]
I do not have any doubts. I just want to create another page (possibly a different app) where I can see all activities of all the tenants. In an admin panel showing the total count of teachers, students, and parents per school.

**Reply by nelson8013** [reply]
@masterix21 Please do you know how I might go about achieving this? If you want, I can share additional detail to help you guide me. In the unlikely event that you cannot help me, please kindly refer me to someone within the community that can assist me. 

Thanks In advance Luca

**Comment by bayangan** [comment, marked as answer]
You only need to loop all the tenants with 
```
Tenant::all()->eachCurrent(function(Tenant $tenant) { 
  //here $tenant is make current, so the switching db is already done
  //you can use query against db to mine data, for eg querying teachers table which exist in tenant db
});
```

**Comment by nelson8013** [comment]
Thank you so much bayangan. This has been very helpful.

**Reply by masterix21** [reply]
IMHO the use of a stats counter is better. Take a look:
https://github.com/spatie/laravel-stats

**Comment by nelson8013** [comment]
Thanks, I'll check it out.
